ReleaseProcessAutomation/Steps/PipelineSteps/ReleaseWithRcStep.cs

```diff
@@ -100,12 +100,12 @@ public void Execute (bool pauseForCommit, bool noPush, string ancestor)
     if (ancestor.Equals("develop"))
     {
       _log.Debug("Getting next possible jira versions for develop from '{nextVersion}'", nextVersion);
```

Comment:
Likely a follow-up ticket: There's two steps `ReleaseWithRcStep` and ReleaseRCStep`. 
* casing in names
* what's the difference between those types? The docs seem similar, the naming, too, and this branch-logic here as well?.
* The one difference is that one type checks just develop and the other develop and release-branch. 

=> maybe just needs better naming? Maybe there's another problem, too?

Reply:
Created a task for evaluation of uses and naming